### PR TITLE
feat: add text translations 

### PR DESCRIPTION
**Pull request by amt-augustineopokukesse**
Added code translations
Refactored template texts for dynamic text rendering

### CODE REVIEW

**Review by Bilal-Abubakari — APPROVED** [review]
Please resolve all comments before merging

File: apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  links = [
+    { name: 'MODEL', url: 'model' },
+    { name: 'MATERIAL', url: 'material' },
+    { name: 'POST-PROCESSING', url: 'postprocessing' },
+    { name: 'PRODUCIBILITY', url: 'producibility' },
+  ];
+  activeLink = this.links[0].name;
+
+  constructor(private translate: TranslateService, private langService: LangService) {
+    this.langService.lang$.subscribe((lang) => {
```
Comment (Nicholas-Darko-Brown) [inline apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts:33]: You should probably consider unsubscribing :)

File: apps/iglidur-designer/src/app/pages/model/model.component.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                font-size: 2rem;
+                font-weight: 500;
+            }
+        }
+        .image {
+            width: 50%;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            // background-color: aqua;
```
Comment (Nicholas-Darko-Brown) [inline apps/iglidur-designer/src/app/pages/model/model.component.scss:24]: Remove all unused code

File: apps/iglidur-designer/src/app/pages/model/model.component.ts [hunk 3]
```diff
@@ -1,11 +1,25 @@
 import { Component } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { LangService } from '@iglidur-designer/services';
+// import tools from '../../../assets/images/tools.png';
```
Comment (Nicholas-Darko-Brown) [inline apps/iglidur-designer/src/app/pages/model/model.component.ts:5]: Same here

File: apps/iglidur-designer/src/app/pages/model/model.component.ts [hunk 4]
```diff
@@ -1,11 +1,25 @@
...
+  imports: [CommonModule, TranslateModule],
   templateUrl: './model.component.html',
   styleUrl: './model.component.scss',
 })
-export class ModelComponent {}
+export class ModelComponent {
+  lang!: string;
+  tools = '../../../assets/images/tools.png';
+  constructor(private translate: TranslateService, private langService: LangService) {
+    this.langService.lang$.subscribe((lang) => {
```
Comment (Nicholas-Darko-Brown) [inline apps/iglidur-designer/src/app/pages/model/model.component.ts:18]: Unsubscribe?

File: apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      this.lang = lang;
+      this.translate.use('toolbar.component.i18n');
```
Comment (Bilal-Abubakari) [inline apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts:35]: Come to think of, does it mean that you will specify translation like this for every component?

File: apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.ts [hunk 6]
```diff
@@ -2,14 +2,46 @@ import { Component } from '@angular/core';
...
+    MatFormFieldModule,
+    MatSelectModule,
+    FormsModule
+  ],
   templateUrl: './header.component.html',
   styleUrl: './header.component.scss',
 })
 export class HeaderComponent {
-  igusLogo = 'assets/igus-logo.svg';
+  lang!: string;
```
[the remark this answers is not in the document]
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.ts:28]: Sure.. I have fixed it.

File: apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts (hunk 1 above)
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts:33]: I have refactored the code

File: apps/iglidur-designer/src/app/pages/model/model.component.ts (hunk 4 above)
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/pages/model/model.component.ts:18]: Done

File: apps/iglidur-designer/src/assets/i18n/ja-JP.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{
```
[the remark this answers is not in the document]
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/assets/i18n/ja-JP.json]: Yes I have

**Pull request merged**